porting/halium_7-1/Configuring.rst

```diff
@@ -118,6 +118,218 @@ At the end of the file, append this::
+
+Bluetooth backporting steps
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The steps are as follows::
+
+    1. Record bluetooth driver and settings. 
+    2. Clone/download the backports scripts.
+    3. Clone/download the kernel source from the newer kernel version you wish to backport from.
+    4. Run the script to integrate the newer sources into your kernel source tree.
+    5. Fix errors that *will* occur when the script is run.
+    6. Add a security patch.
+    7. Make necessary changes to your kernel defconfig file.
+    8. Rebuild and flash the full halium-boot.img.
+    9. Add bluetooth initializaton script, rebuild and flash system.img.
+
+.. _BT-driver:
+
+Determine driver and bluetooth settings for your device
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By the time you reach this point in the porting process, you will have completed building halium-boot (probably a number of times). It is wise to note all kernel defconfig settings related to bluetooth before proceeding with the steps below. One of these settings designates the bluetooth driver used by your device, which you will need to know at a later stage.
+
+Although many of these settings will be collected in one place in your defconfig file, some will likely be in other places as well, making it difficult to ascertain all necessary settings. The directions below may help.
+
+After completing a build of halium-boot.img, go to your ``out/target/product/[device]/obj/KERN_OBJ`` directory. There, run the following command::
+
+    ARCH=arm64 make menuconfig
+
+If your device is armhf, use ``ARCH=arm`` instead.
+
+This will bring up menuconfig complete with the defconfig settings from your build. You then navigate to the bluetooth drivers submenu and browse through all activated settings, recording which ``CONFIG_xxxxx`` settings apply to those that are activated for your device, as well as the information about what this setting does and which other settings it depends upon (found under Help). Save this information for later reference.
+
+Download the backport scripts
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Clone the backport scripts into a directory outside your halium source tree by issuing this command from your home (~) directory::
+
+    git clone https://github.com/ubuntu-phonedations/backports.git -b for-ubuntu backport-scripts
+
+This downloads the backport scripts prepared by Canonical based on the :ref:`original Backports Project <Backports>` mentioned above, and places them in the directory ``~/backport-scripts``. The scripts are specifically written to backport from kernel version 4.2.
+
+Download kernel source to backport from
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Create a directory (outside your halium source tree) for the kernel source from which you will pull the newer drivers::
+
+    mkdir ~/kernel-backports
+
+Now clone the kernel source for the version and branch you need (v4.2 in the example below) into this directory::
```

Comment:
I suggest to remove the '(v4.2 in the example below)'

it's not just an example. as you explain a few lines below and above it's the only option

We could just say: 'Now clone the kernel source for v4.2.'

Reply:
Yes, that is a remnant from a different text I had to change.